libs/datamodel/core/tests/attributes/default_positive.rs

```diff
@@ -45,6 +48,16 @@ fn should_set_default_for_all_scalar_types() {
         .assert_default_value(DefaultValue::new_single(PrismaValue::DateTime(
             DateTime::parse_from_rfc3339("2019-06-17T14:20:57Z").unwrap(),
         )));
+    user_model
+        .assert_has_scalar_field("bytes")
+        .assert_base_type(&ScalarType::Bytes)
+        .assert_default_value(DefaultValue::new_single(PrismaValue::Bytes(b"hello world".to_vec())));
+    user_model
+        .assert_has_scalar_field("json")
+        .assert_base_type(&ScalarType::Json)
+        .assert_default_value(DefaultValue::new_single(PrismaValue::Json(
+            r#"{ "a": ["b"] }"#.to_owned(),
+        )));
 }
```

Comment:
decimal is not validated like the other two

Reply:
good catch, done